packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -334,4 +374,108 @@ describe('Model > TutorialStore', function () {
       }).then(done, done)
     })
   })
+
+  describe('Actions > resetSeen', function () {
+    it('should reset only the tutorial seen time if tutorial is the kind', function () {
```

Comment:
'should only reset the tutorial seen time for tutorials.'

Reply:
I'm trying to be clear between mini-course kind and tutorial kind of tutorial resources. I don't think saying just 'tutorials' is specific enough. I can definitely reword this though.